specification.md

```diff
@@ -250,6 +277,46 @@ In OpenTracing we force `SpanContext` instances to be **immutable** in order to
 
 This is modeled in different ways depending on the language, but semantically the caller should be able to efficiently iterate through all baggage items in one pass given a `SpanContext` instance.
 
+### `ScopeManager`
+
+The `ScopeManager` interface sets and retrieves the active `Span`, working along the `Scope` container interface.
```

Comment:
Related to my previous comment. Here I guess the behavior is clear, but I think we may need to change something in the other paragraph.